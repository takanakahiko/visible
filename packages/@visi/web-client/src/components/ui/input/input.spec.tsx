import { render } from '@testing-library/react';
import React from 'react';

import { Input } from '.';

describe('Input', () => {
  it('matches snapshot', () => {
    const { container } = render(<Input />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
