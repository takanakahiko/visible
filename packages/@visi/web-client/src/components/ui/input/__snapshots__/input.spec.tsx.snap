// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input matches snapshot 1`] = `
<input
  class="border-gray-500 bg-gray-100 border placeholder-gray-600 text-sm py-2 px-4 w-64 focus:outline-none focus:shadow-outline focus:border-primary-500 focus:bg-white disabled:opacity-75 disabled:cursor-not-allowed rounded-full"
/>
`;
